logstash.go

```diff
@@ -56,8 +57,8 @@ func (a *LogstashAdapter) Stream(logstream chan *router.Message) {
 		}
 		_, err = a.conn.Write(js)
 		if err != nil {
-			log.Println("logstash:", err)
-			continue
+			log.Println("fatal logstash:", err)
```

Comment:
It would be more clear to use `log.Fatalln()` (exit signal 1), is there a reason for exit signal 3?